website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -22,42 +22,36 @@ Admin partitions exist a level above namespaces in the identity hierarchy. They
 
 ### Default Admin Partition
 
-Each Consul cluster will have at least one default admin partition (named `default`). Any resource created without specifying an admin partition will inherit the partition of the ACL token.
+Each Consul cluster will have a default admin partition (named `default`). The `default` admin partition is special because it may contain namespaces and other entities that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
 
-The `default` admin partition is special in that it may contain namespaces and other entities that are replicated between datacenters. The `default` partition must also contain the Consul servers.  
+Any resource created without specifying an admin partition will inherit the partition of the ACL token.
 
 -> **Preexisting resources and the `default` partition**: Admin partitions were introduced in Consul 1.11. After upgrading to Consul 1.11 or later, the `default` partition will contain all resources created in previous versions.
 
 ### Naming Admin Partitions
 
 Only characters that are valid in DNS names can be used to name admin partitions.
-Names must also start with a letter.
+Names must also begin with a lowercase letter.
 
 ### Namespaces
 
-When an admin partition is created, it will include a `default` namespace. You can create additional namespaces within the partition.
-
-All namespaces must exist within an admin partition. By extension, the partition will also contain all namespaced resources.
-
-Within a single datacenter, a namespace in one admin partition is logically separate from any other namespace with the same name in other admin partitions.
+When an admin partition is created, it will include the `default` namespace. You can create additional namespaces within the partition. Resources created within a namespace will be inherited by the partition.
```

Comment:
> `Resources created within a namespace will be inherited by the partition`

The phrasing of this statement is a little surprising to me. Given partitions contain namespaces, defining it as inheritance seems incorrect. `Resources created within a namespace are not shared across partitions` maybe?